de.dashup.application/src/main/webapp/views/marketplace.jsp

```diff
@@ -7,17 +9,136 @@
+                <div class="col s12">
+                    <a href="${pageContext.request.contextPath}/" class="breadcrumb">dashup</a>
+                    <a href="#" class="breadcrumb"><fmt:message key="i18n.marketplace"/></a>
+                </div>
+            </div>
+        </nav>
+        <form action="${fn:escapeXml(pageContext.request.contextPath)}/marketplace/search" method="post">
+            <div class="row" style="margin-top: 25px">
+                <div class="col m4 offset-m4 s8 offset-s1">
+                    <div class="input-field">
+                        <input id="text-field-login-email" name="email" type="text" class="validate">
+                        <label for="text-field-login-email"><fmt:message key="i18n.enterSearchterm"/></label>
+                    </div>
+                </div>
+                <div class="col m2 s2" style="margin-top: 18px">
+                    <button id="btn_start_search_marketplace" class="btn waves-effect waves-light" type="submit"
+                            name="search">
+                        <fmt:message key="i18n.go"/>
+                    </button>
+                </div>
+            </div>
+        </form>
+        <div class="row">
+            <div class="col m6 offset-m3 s10 offset-s1">
+                <h3><fmt:message key="i18n.featured"/></h3>
+                <hr/>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col m6 offset-m3 s10 offset-s1">
+                <div class="carousel carousel-slider center">
+                    <div class="carousel-item red white-text" href="#one!">
+                        <h2>First Panel</h2>
+                        <p class="white-text">This is your first panel</p>
+                    </div>
+                    <div class="carousel-item amber white-text" href="#two!">
+                        <h2>Second Panel</h2>
+                        <p class="white-text">This is your second panel</p>
+                    </div>
+                    <div class="carousel-item green white-text" href="#three!">
+                        <h2>Third Panel</h2>
+                        <p class="white-text">This is your third panel</p>
+                    </div>
+                    <div class="carousel-item blue white-text" href="#four!">
+                        <h2>Fourth Panel</h2>
+                        <p class="white-text">This is your fourth panel</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col m6 offset-m3 s10 offset-s1">
+                <h3><fmt:message key="i18n.mostPopular"/></h3>
+                <hr/>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col s12 m3 offset-m3">
+                <div class="card horizontal">
+                    <div class="card-image">
+                        <img src="https://lorempixel.com/100/190/nature/6">
```

Comment:
I should ask @raphaelmue whether he could add a feature to select an icon for the widget before publishing